<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page0">
     <widget class="QPushButton" name="budgetMenuButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>281</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Budget Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="menuButton2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>270</y>
        <width>281</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>menu button 2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page1">
     <widget class="QPushButton" name="newBudgetButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>New Budget</string>
      </property>
     </widget>
     <widget class="QPushButton" name="budgetListButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>270</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Budget List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnFromBudgetMenuButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QPushButton" name="cancelCreateBudgetButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>290</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createNewBudgetButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>290</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="newBudgetNameInput">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="newBudgetNameLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Budget Name:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>691</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="addItemLayOut">
       <item row="0" column="0">
        <widget class="QLabel" name="ItemNameLabel">
         <property name="text">
          <string>Item Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="itemNameInput"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="LimitLabel">
         <property name="text">
          <string>Limit:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="itemLimitInput"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ItemListLabel">
         <property name="text">
          <string>Item  List:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTableWidget" name="itemTableWidget">
         <column>
          <property name="text">
           <string>Item Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Item total</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Category</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="finishAddItemButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>500</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>finish</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addItemButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>500</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QPushButton" name="addItemToBudgetButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Item</string>
      </property>
     </widget>
     <widget class="QListWidget" name="budgetListWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>491</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="budgetListLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Budget List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteBudgetButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editBudgetButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnFromBudgetListButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>500</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>return</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page5"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
